From dd5bed5198bda65930d4ea406e29006f0deb6018 Mon Sep 17 00:00:00 2001
From: Marcus Calhoun-Lopez <marcuscalhounlopez@gmail.com>
Date: Sun, 22 Nov 2015 10:24:04 -0700
Subject: [PATCH] OS X: resolve SDK version of QMAKE_LINK_C and
 QMAKE_LINK_C_SHLIB

On OS X, both compilers and linkers are given an absolute path.
For consistency, the same should be done with the C linkers.
The change is also a convenience to the MacPorts project,
which actively discourages ambiguous compiler names.
(https://trac.macports.org/wiki/UsingTheRightCompiler).

Change-Id: Ic1885aed825340696e9fde766788eebf51de3ff6
---
 mkspecs/features/mac/sdk.prf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkspecs/features/mac/sdk.prf b/mkspecs/features/mac/sdk.prf
index e238f8e..624366a 100644
--- mkspecs/features/mac/sdk.prf
+++ mkspecs/features/mac/sdk.prf
@@ -31,7 +31,7 @@
 QMAKESPEC_NAME = $$basename(QMAKESPEC)
 
 # Resolve SDK version of various tools
-for(tool, $$list(QMAKE_CC QMAKE_CXX QMAKE_FIX_RPATH QMAKE_AR QMAKE_RANLIB QMAKE_LINK QMAKE_LINK_SHLIB QMAKE_ACTOOL)) {
+for(tool, $$list(QMAKE_CC QMAKE_CXX QMAKE_FIX_RPATH QMAKE_AR QMAKE_RANLIB QMAKE_LINK QMAKE_LINK_SHLIB QMAKE_ACTOOL QMAKE_LINK_C QMAKE_LINK_C_SHLIB)) {
     tool_variable = QMAKE_MAC_SDK.$${QMAKESPEC_NAME}.$${QMAKE_MAC_SDK}.$${tool}
     !isEmpty($$tool_variable) {
         $$tool = $$eval($$tool_variable)
-- 
2.6.2.2.g1b5ffa3

